contrib/comdb2makecluster

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+
+#set -x
+
+if [[ $1 = "--dir" ]] ; then 
+    shift
+    DBDIR=$1
+    shift
+fi
+
+DBNAME=$1
+shift
+CLUSTER=$@
+
+# use --dir to specify db directory otherwise creates db in current_dir/$DBNAME
+if [[ -z $DBDIR ]]; then
+    DBDIR=`pwd`/$DBNAME/
+fi
+
+failexit() {
+    echo $1
+    exit -1
+}
+
+usage() {
+    failexit "usage: $0 [--dir /path/to/dir] dbname node1 node2 [..nodeN]"
+}
+
+
+
+if [ "x$CLUSTER" = "x" ] ; then
+    usage
+fi
+
+debug=0
+while [[ $# -gt 0 && $1 = -* ]]; do
+    [[ $1 = '-debug' ]] && debug=1
+    shift
+done
+
+TMPDIR=${TMPDIR:-/tmp}
+CDB2_CONFIG=$DBDIR/comdb2db.cfg
+CDB2_OPTIONS="--cdb2cfg ${CDB2_CONFIG}"
+comdb2task=comdb2
+
+which $comdb2task > /dev/null || failexit "$comdb2task not in path"
+which copycomdb2  > /dev/null || failexit "copycomdb2 not in path"
+
+set -e
+
+vars="DBNAME DBDIR TMPDIR CDB2_OPTIONS CDB2_CONFIG"
+for required in $vars; do
+    q=${!required}
+    if [[ -z "$q" ]]; then
+        echo "$required not set" >&2
+        exit 1
+    fi
+#    echo "$required=$q"
+done
+
+# Setup a debugger to run comdb2 server
+DEBUG_PREFIX=
+
+if [[ -n ${DEBUGGER} ]]; then
```

Comment:
Did you want to keep the debug options here?  They're very useful for `make test`, but probably not for this.